.envrc

```diff
@@ -90,13 +83,10 @@ if ! type source_up >/dev/null 2>/dev/null; then
     PWD="$(pwd)"
 fi
 
+FISSILE_DEFAULT_STEMCELL=${FISSILE_STEMCELL:-${FISSILE_DOCKER_REGISTRY}${FISSILE_DOCKER_ORGANIZATION}/fissile-stemcell-sle12:$FISSILE_STEMCELL_VERSION}
```

Comment:
I don't think we want this as it is weird to access `FISSILE_STEMCELL` in the `FISSILE_DEFAULT_STEMCELL` and then export `FISSILE_STEMCELL` with the default value. The point of my previous comment was to make it easier to read, not harder. Instead, do
```
export FISSILE_STEMCELL=${FISSILE_STEMCELL:-$FISSILE_DEFAULT_STEMCELL}
```
and remove `FISSILE_STEMCELL` from the `FISSILE_DEFAULT_STEMCELL`.

Sorry, I was not very explicit the first time.